README.md

```diff
@@ -92,6 +92,39 @@ There are several other implementations available depending on your needs.
 | cce_kahan_full_c_full_e (cce_exact) | This additionally removes gradient filtering from the embedding gradient. This is useful as a reference point/sanity check. |
 
 
+### Vocabulary Parallelism
+
+We also support computing linear cross-entropy loss for classifier weights sharded
+along the vocabulary dimensions. To use this, simply provided a `VocabParallelOptions` instance
+to `linear_cross_entropy`. This takes 3 parameters, the `start` and `stop` indices of this ranks
+shard, and the `torch.distributed.ProcessGroup` that defines the vocab parallel group for this rank.
+
+
+
+```python
+import torch
+
+from cut_cross_entropy import linear_cross_entropy, VocabParallelOptions
+
+vp_group = torch.distributed.new_group(...)
```

Comment:
that's great! Think this `vp_group` can easily be extract from PT's device_mesh object no: https://pytorch.org/tutorials/recipes/distributed_device_mesh.html?highlight=devicemesh ?

Reply:
I updated the readme a little bit less perceptive on how to retrieve the vocab parallel group. The intent here is just to show various different ways you could create/retrieve the vocab parallel group as I don't want to force one particular way on users.